src/distributed/assetstorage.did

```diff
@@ -6,6 +6,10 @@ type Time = int;
 type CreateAssetArguments = record {
   key: Key;
   content_type: text;
+  properties: AssetProperties;
+};
+
+type AssetProperties = record {
```

Comment:
This is a breaking change at the interface level.  Is this intentional?

Reply:
thank you, and no -- I completely forgot to think about this quite important detail
let's discuss this during stand up

Reply:
This is still a breaking change at the interface level.  So at the moment, this PR is more than just "pretty print asset properties during deployment."  Am I forgetting a resolution with regard to this?